sites/site-kit/src/lib/components/Nav.svelte

```diff
@@ -150,7 +150,7 @@
 		line-height: 1;
 	}
 
-	@media (max-width: 800px) {
+	@media (max-width: 799px) {
```

Comment:
this needs to be 800px otherwise this happens 

<img width="874" alt="image" src="https://user-images.githubusercontent.com/1162160/206622048-46c77904-2543-45fe-8dc8-e17e21b23558.png">

```suggestion
	@media (max-width: 800px) {
```

Reply:
Thank you Rich!
However, there's still one niggling problem that only happens in dark mode.

The mobile menu colour is different from the navbar and still sticks at 800px only.
<img width="859" alt="SvelteKit different coloured navbar in dark mode" src="https://user-images.githubusercontent.com/54401897/206624597-6f16acd9-f999-4503-8fb1-9f93b0390d96.png">

Is there a quick way to add this fix to remove the mobile menu background colour or should I open another PR?
<img width="371" alt="remove mobile background colour from Nav component" src="https://user-images.githubusercontent.com/54401897/206625065-199750d9-1c7b-4503-9f2f-ebe68d6440eb.png">